README.md

```diff
@@ -43,9 +43,25 @@ spec:
   serviceNetwork:
   - 172.30.0.0/16
 ```
+Alternatively, OVN can be configured as follows:
```

Comment:
I don't think it's useful to repeat the entire config object with only one line changed. Just say something like "Alternatively, OVN could be chosen by specifying `networkType: OVNKubernetes`". (And drop the converted operator config below entirely.)

Although also, we need to figure out what the deal is with README vs Tech Preview-ness, etc. We don't want too much OVN-pimping at this point.

I guess Kuryr is going to have the same problem... maybe we should have `README.OVN.md` and `README.Kuryr.md`? Then we can go into as much detail as we want without derailing the main "narrative".